frontend/src/views/ensemble/Barline.tsx

```diff
@@ -2,29 +2,29 @@ import { Box } from "@mui/material";
 import _ from "lodash";
 import SingleNote from "./SingleNote";
 
-// consts for calculating board, not including volume bar
-const boardWidth = 32; // 8 bars of 4 quarter notes = 32 note placements
-const boardHeight = 8; // 8 Notes from C -> C
+// consts for calculating barline, not including volume bar
+const barlineWidth = 32; // 8 bars of 4 quarter notes = 32 note placements
```

Comment:
We could probably name this with the all caps convention, since they are true constants. 

`BARLINE_WIDTH` and `BARLINE_HEIGHT`.

Reply:
At some point, we probably just want this to be determined dynamically by reading the 2D matrix in `Ensemble`. This could allow us to make a change later, if we want to, for example, add more columns for a wider barline.